### PR TITLE
Add ticket number to trailblazer

### PR DESCRIPTION
**Pull request by GustavLagneborgScilifelab**
## Description
Adding ticket number to trailblazer

### Added

- Adding ticket number to the request body sent from CG to trailblazer

### Changed

-

### Fixed

-

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_main -t cg -b [THIS-BRANCH-NAME] -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: cg/apps/tb/api.py [hunk 1]
```diff
@@ -148,7 +148,9 @@ def add_pending_analysis(
         slurm_quality_of_service: SlurmQos,
         email: str = None,
         data_analysis: Pipeline = None,
+        ticket: str = None,
```
[the remark this answers is not in the document]
Comment (henrikstranneheim) [reply cg/apps/tb/api.py:151]: Should we call it `ticket_id`? Just `ticket` sounds like an object to me or at least more than just the id

**Review by henrikstranneheim — APPROVED** [review]
👍

File: cg/apps/tb/api.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (GustavLagneborgScilifelab) [reply cg/apps/tb/api.py:151]: As it is a string in StatusDB, I wanted to keep the same logic. I renamed the ticket to ticket_id in trailblazer as it's a new database column. But for CG, it's named ticket everywhere else, so introducing a name change might open up a lot of refactoring.

[the remark this answers is not in the document]
Comment (henrikstranneheim) [reply cg/apps/tb/api.py:151]: Fair enough, then let's save that for a focused PR later and let´s keep `ticket` in cg and let's go with string to keep it consistent

**Comment by GustavLagneborgScilifelab** [comment]
Deployed in production: 
Screenshot 2022-11-14 at 15 14 35

**Pull request merged**